py/kubeflow/kubeflow/ci/workflow_utils.py

```diff
@@ -201,6 +201,12 @@ def create_kaniko_task(self, task_template, dockerfile, context,
         """
         kaniko = argo_build_util.deep_copy(task_template)
 
+        # append the tag base-commit[0:7]
+        if ":" not in destination:
+            sha = os.getenv("PULL_BASE_SHA", "12341234kanikotest")
+            base = os.getenv("PULL_BASE_REF", "master")
+            destination += ":%s-%s" % (base, sha[0:8])
```

Comment:
I think you made a typo here. I believe it should be sha[0:7] and not sha[0:8]. The comment does state sha[0:7].

Reply:
I didn't calculate correctly in the message above, I'll comment there as well.

I had in mind to have the same length it previously had https://console.cloud.google.com/gcr/images/kubeflow-images-public/GLOBAL/jupyter-web-app?gcrImageListsize=30

Reply:
I was going with the first 7 characters as this is the Commit ID you see in GitHub. This way, it is easy for users to track down what commit and image originates from (and from which branch). 
![image](https://user-images.githubusercontent.com/28541758/110160965-ffa9bd00-7dec-11eb-9725-04134288cb02.png)

Reply:
Resolving this one and continuing the discussion on the above conversation